lwk_wollet/src/clients/asyncr/esplora.rs

```diff
@@ -646,6 +661,12 @@ impl EsploraClientBuilder {
         self
     }
 
+    /// Set the concurrency level for requests, default is 1
+    pub fn concurrency(mut self, concurrency: usize) -> Self {
```

Comment:
what if `concurrency` is 0?

from https://docs.rs/futures it seems fine

Reply:
It seems using 0 breaks the test...

Reply:
Using 1 if 0 is passed

https://github.com/Blockstream/lwk/pull/84/commits/c4b180e68ec93539ca6ad1cef7edbd5db627a33e